src/platforms/android/include.gradle

```diff
@@ -1,10 +1,6 @@
 //default elements
 android {
-  productFlavors {
-    "nativescript-background-http" {
-      dimension "nativescript-background-http"
-    }
-  }
+  
```

Comment:
If the product flavor is removed, shouldn't the {N} version in src/package.json be increased? I don't think 3.0.0 will work with the updated plugin.